test/alternator/test_metrics.py

```diff
@@ -174,7 +214,7 @@ def test_batch_get_item_count(test_table_s, metrics):
 
 KB = 1024
 def test_rcu(test_table_s, metrics):
-    with check_increases_metric_exact(metrics, 'scylla_alternator_rcu_total', [[2, None]]):
+    with check_increases_metric_exact(metrics, 'scylla_alternator_table_rcu_total', [[2, {'cf': test_table_s.name}]]):
```

Comment:
Why did you replace the total test by a per-table test instead of having both?
You **did** keep both the total counters and per-table counters, right? That's what I thought you understood...
If you kept both I think the test should check both, no?

Reply:
Oh, maybe just the RCU is only per-table and we don't keep a total counter?

Reply:
That was my original plan, but I've changed my mind and V2 has both